From: Benjamin Drung <bdrung@debian.org>
Date: Thu, 28 Dec 2017 16:26:37 +0000
Subject: Wrap long lines in man pages

Some man pages provoke warnings or errors from man.
.
"cannot adjust" or "can't break" indicate trouble with paragraph filling,
usually related to long lines. Adjustment can be helped by left
justifying, breaks can be helped with hyphenation, see "Manipulating
Filling and Adjusting" and "Manipulating Hyphenation" in the groff
manual (see info groff).
.
To test this for yourself you can use the following command:
 LC_ALL=en_US.UTF-8 MANROFFSEQ='' MANWIDTH=80 \
         man --warnings -E UTF-8 -l -Tutf8 -Z <file> >/dev/null
.
Refer to the groff_man(7) manual page and the groff_mdoc(7) manual
page for details.
---
 man/man3/fi_getinfo.3 | 5 +++--
 man/man3/fi_trigger.3 | 5 +++++
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/man/man3/fi_getinfo.3 b/man/man3/fi_getinfo.3
index 84f0a6d..4b14420 100644
--- a/man/man3/fi_getinfo.3
+++ b/man/man3/fi_getinfo.3
@@ -11,7 +11,8 @@ fi_allocinfo, fi_dupinfo \- Allocate / duplicate an fi_info structure
 #include\ <rdma/fabric.h>
 
 int\ fi_getinfo(int\ version,\ const\ char\ *node,\ const\ char\ *service,
-\ \ \ \ \ \ \ \ uint64_t\ flags,\ const\ struct\ fi_info\ *hints,\ struct\ fi_info\ **info);
+\ \ \ \ \ \ \ \ uint64_t\ flags,\ const\ struct\ fi_info\ *hints,
+\ \ \ \ \ \ \ \ struct\ fi_info\ **info);
 
 void\ fi_freeinfo(struct\ fi_info\ *info);
 
@@ -619,7 +620,7 @@ with their GNI protocol.
 \f[I]FI_ADDR_STR\f[] : Address is a formatted character string.
 The length and content of the string is address and/or provider
 specific, but in general follows a URI model:
-.IP
+.PP
 .nf
 \f[C]
 address_format[://[node][:[service][/[field3]...][?[key=value][&k2=v2]...]]]
diff --git a/man/man3/fi_trigger.3 b/man/man3/fi_trigger.3
index 10a2d29..2f2d5e7 100644
--- a/man/man3/fi_trigger.3
+++ b/man/man3/fi_trigger.3
@@ -80,8 +80,13 @@ The threshold is specified using struct fi_trigger_threshold:
 .nf
 \f[C]
 struct\ fi_trigger_threshold\ {
+<<<<<<< HEAD
 \ \ \ \ struct\ fid_cntr\ *cntr;\ /*\ event\ counter\ to\ check\ */
 \ \ \ \ size_t\ threshold;\ \ \ \ \ \ /*\ threshold\ value\ */
+=======
+\ \ \ \ \ struct\ fid_cntr\ *cntr;\ \ \ \ \ /*\ event\ counter\ to\ check\ */
+\ \ \ \ \ size_t\ threshold;\ \ \ \ \ \ \ \ \ \ /*\ threshold\ value\ */
+>>>>>>> d4bdac0... Wrap long lines in man pages
 };
 \f[]
 .fi
